# request
std_msgs/String command
---
# response
std_msgs/Bool done
